README.md

```diff
@@ -1,79 +1,27 @@
 # goharbor-client
 
-![Compile & Test](https://github.com/mittwald/goharbor-client/workflows/Compile%20&%20Test/badge.svg)
-[![GitHub license](https://img.shields.io/github/license/mittwald/goharbor-client.svg)](https://github.com/mittwald/goharbor-client/blob/master/LICENSE)
-[![Go Report Card](https://goreportcard.com/badge/github.com/mittwald/goharbor-client)](https://goreportcard.com/report/github.com/mittwald/goharbor-client)
-[![Documentation](https://godoc.org/github.com/mittwald/goharbor-client?status.svg)](https://pkg.go.dev/github.com/mittwald/goharbor-client)
+[![GitHub license](https://img.shields.io/github/license/mittwald/goharbor-client.svg?style=flat-square)](https://github.com/mittwald/goharbor-client/blob/master/LICENSE)
+[![Go Report Card](https://goreportcard.com/badge/github.com/mittwald/goharbor-client?style=flat-square)](https://goreportcard.com/badge/github.com/mittwald/goharbor-client)
+[![Go Doc](https://img.shields.io/badge/godoc-reference-blue.svg?style=flat-square)](https://pkg.go.dev/github.com/mittwald/goharbor-client)
+[![Release](https://img.shields.io/github/release/mittwald/goharbor-client.svg?style=flat-square)](https://github.com/mittwald/goharbor-client/releases/latest)
 
-<hr />
+[![Actions Status](https://github.com/mittwald/goharbor-client/workflows/Verify%20code%20generation%20integrity/badge.svg)](https://github.com/mittwald/goharbor-client/actions)
+[![Actions Status](https://github.com/mittwald/goharbor-client/workflows/Test/badge.svg)](https://github.com/mittwald/goharbor-client/actions)
+---
 
 :warning: **STABILITY NOTICE:** This project is still under development and not stable yet - breaking changes may happen at any time and without notice
 
-<hr/>
+---
 
-A Harbor API client enabling Go programs to perform CRUD operations on [goharbor](https://github.com/goharbor/harbor) users and projects.
+A Go client library enabling programs to perform CRUD operations on the [goharbor](https://github.com/goharbor/harbor) API.
 
-This library is built upon `goharbor/v1.10.2`, and utilizes typings from the upstream source [goharbor/harbor](https://github.com/goharbor/harbor), available under the 
-[Apache 2 license](https://github.com/goharbor/harbor/blob/master/LICENSE).
-
-The initial project is a fork of [TimeBye/go-harbor](https://github.com/TimeBye/go-harbor) and available under the MIT License.
+This client library utilizes types generated by [go-swagger](https://github.com/go-swagger/go-swagger).
 
 ## Installation
 
 Install this library using `go get`:
 
     $ go get github.com/mittwald/goharbor-client
 
-## Usage
-
-Initialize a new goharbor client, then use the various services on the client to
-access different parts of the Harbor API.
-
-```go
-package main
-import (
-	"errors"
-	"fmt"
-	harbor "github.com/mittwald/goharbor-client"
-)
-
-func main() {
-	client, err := harbor.NewClient("url", "username", "password")
```

Comment:
Maybe add an [example function](https://blog.golang.org/examples) instead of dropping the example entirely?

Reply:
Good point. I've added a very basic example into `client_test.go`, which the `README` now refers to. With mocked client interfaces coming up in the near future, we can write both unit tests and detailed example functions more easily.